package-res/resources/web/pentaho/visual/ccc/abstract/View.js

```diff
@@ -21,22 +21,24 @@ define([
   "cdf/lib/CCC/protovis",
   "../_axes/Axis",
   "../util",
+  "pentaho/util/object",
+  "pentaho/data/filter",
   "pentaho/visual/events",
   "pentaho/visual/color/utils",
   "pentaho/visual/color/paletteRegistry",
   "pentaho/data/TableView",
   "pentaho/i18n!view"
 ], function(View,
-    def, pvc, cdo, pv, Axis,
-    util, visualEvents, visualColorUtils, visualPaletteRegistry,
-    DataView, bundle) {
+            def, pvc, cdo, pv, Axis,
```

Comment:
This was obviously an accident, in the unlikely case you did not notice. The tone of the comment is unnecessary and harms the reviewing process.

If reviewing partial work is too much of a strain, then we need to postpone putting up pull requests for RFC purposes until the code is in a better shape and revise the overall reviewing strategy.